taxcalc/tests/test_policy.py

```diff
@@ -36,6 +36,13 @@ def test_incorrect_Policy_instantiation():
         p = Policy(gfactors=dict())
     with pytest.raises(ValueError):
         p = Policy(parameter_dict=list())
+    # similar test to test_behavior.test_incorrect_behavior_instantiation-2
+    # fails because this dict is not validated
+    bad_pol_dict = {
+        '_SS_Earnings_c': {'start_year': 2013, 'value': [0.1]}
+    }
+    with pytest.raises(ValueError):
+        p = Policy(parameter_dict=bad_pol_dict)
     with pytest.raises(ValueError):
```

Comment:
@martinholmer sorry for the confusion. I am referring to the [second test case](https://github.com/open-source-economics/Tax-Calculator/blob/master/taxcalc/tests/test_behavior.py#L14-L18) in `test_behavior.test_incorrect_behavior_instantiation`:

```
    bad_behv_dict = {
        '_BE_bad': {'start_year': 2013, 'value': [0.0]}
    }
    with pytest.raises(ValueError):
        Behavior(behavior_dict=bad_behv_dict)
```